modules/search/instant-search/components/search-app.jsx

```diff
@@ -156,13 +144,16 @@ class SearchApp extends Component {
 		this.showResults();
 	};
 
+	handleOverlayTriggerClick = event => {
+		event.preventDefault();
+		event.stopImmediatePropagation();
```

Comment:
Looks like this may not be having the intended effect - the search box still opens in Twenty Twenty in the background:

<img width="1726" alt="Screen Shot 2020-08-07 at 11 44 05" src="https://user-images.githubusercontent.com/17325/89593608-b10e6180-d8a3-11ea-80d4-86198304ae0c.png">

Reply:
This should be fixed 🤞